backend/routes/forms/form.py

```diff
@@ -39,6 +40,35 @@ async def get(self, request: Request) -> JSONResponse:
 
         return JSONResponse({"error": "not_found"}, status_code=404)
 
+    @requires(["authenticated", "admin"])
+    @api.validate(
+        resp=Response(
+            HTTP_200=OkayResponse,
+            HTTP_400=ErrorMessage,
+            HTTP_404=ErrorMessage
+        ),
+        tags=["forms"]
+    )
+    async def patch(self, request: Request) -> JSONResponse:
+        """Updates form by ID."""
+        data = await request.json()
+
+        if raw_form := await request.state.db.forms.find_one(
+            {"_id": request.path_params["form_id"]}
+        ):
+            if "_id" in data or "id" in data:
+                return JSONResponse({"error": "locked_field"}, status_code=400)
+
+            raw_form.update(data)
+            try:
+                form = Form(**raw_form)
+            except ValidationError as e:
+                return JSONResponse(e.errors(), status_code=400)
```

Comment:
Our new policy for Pydantic validation errors is HTTP 422, since this will automatically pick up the response structure for pydantic validations.

```suggestion
                return JSONResponse(e.errors(), status_code=422)
```